frontend/src/components/pages/ProfileCompletionThree.jsx

```diff
@@ -0,0 +1,340 @@
+  };
+  const handleLocationType = () => {
+    setOpenLocationType(!openLocationType);
+    setOpenEmployment(false);
+  };
+  const handleToggleEndDate = () => {
+    setEndDateVisible(!endDateVisible);
+  };
+  const handleStartMonth = () => {
+    setOpenStartMonth(!openStartMonth);
+    setOpenEndYear(false);
+    setOpenEndMonth(false);
+    setOpenStartYear(false);
+    setOpenLocationType(false);
+  };
+  const handleEndMonth = () => {
+    setOpenEndMonth(!openEndMonth);
+    setOpenEndYear(false);
+    setOpenStartMonth(false);
+    setOpenStartYear(false);
+    setOpenLocationType(false);
+  };
+  const handleStartYear = () => {
+    setOpenStartYear(!openStartYear);
+    setOpenEndYear(false);
+    setOpenEndMonth(false);
+    setOpenStartMonth(false);
+  };
+  const handleEndYear = () => {
+    setOpenEndYear(!openEndYear);
+    setOpenEndMonth(false);
+    setOpenStartMonth(false);
+    setOpenStartYear(false);
+  };
+  const Months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+
+  // Pushes every year from 1930 to 2023 to years list
+  const Years = [];
+  const max = 2023;
+  const min = 1930;
+  for (let i = max; i >= min; i--) {
+    Years.push(i);
+  }
+
+  // These functions gets the text-content when element children is clicked
+  // It stores the value into DataValue
+
+  const handleEmploymentClick = (event) => {
+    const EmploymentDataValue = event.currentTarget.textContent;
+    const select = document.querySelector(".Employment-type-text");
+    select.textContent = EmploymentDataValue;
+    setOpenEmployment(false);
+  };
+
+  const handleLocationClick = (event) => {
+    const LocationDataValue = event.currentTarget.textContent;
+    const select = document.querySelector(".Location-text");
+    select.textContent = LocationDataValue;
+    setOpenLocationType(false);
+  };
+
+  const handleStartMonthClick = (event) => {
+    const StartMonthDataValue = event.currentTarget.textContent;
+    const select = document.querySelector(".Start-Month");
+    select.textContent = StartMonthDataValue;
+    setOpenStartMonth(false);
+  };
+
+  const handleStartYearClick = (event) => {
+    const StartYearDataValue = event.currentTarget.textContent;
+    const select = document.querySelector(".Start-Year");
+    select.textContent = StartYearDataValue;
+    setOpenStartYear(false);
+  };
+  const handleEndMonthClick = (event) => {
+    const EndMonthtDataValue = event.currentTarget.textContent;
+    const select = document.querySelector(".End-Month");
+    select.textContent = EndMonthtDataValue;
+    setOpenEndMonth(false);
+  };
+
+  const handleEndYearClick = (event) => {
+    const EndYearDataValue = event.currentTarget.textContent;
+    const select = document.querySelector(".End-Year");
+    select.textContent = EndYearDataValue;
+    setOpenEndYear(false);
+  };
```

Comment:
näissäkään ei ole mitään muuta eroa kuin classin nimi, mikset vaan korvaa kaikkia näitä tällä
```js
  const handleEndYearClick = (event, class) => {
    const EndYearDataValue = event.currentTarget.textContent;
    const select = document.querySelector(class);
    select.textContent = EndYearDataValue;
    setOpenEndYear(false);
  };
```

Reply:
Korjaan huomen aamust